src/FormControl/FormControl.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react'
+import {Autocomplete, Box, Checkbox, Radio, Select, Textarea, TextInput, TextInputWithTokens, useSSRSafeId} from '..'
+import InputValidation from '../_InputValidation'
+import {ComponentProps} from '../utils/types'
+import FormControlCaption from './_FormControlCaption'
+import FormControlLabel from './_FormControlLabel'
+import FormControlValidation from './_FormControlValidation'
+import {Slots} from './slots'
+import ValidationAnimationContainer from '../_ValidationAnimationContainer'
+import {get} from '../constants'
+import FormControlLeadingVisual from './_FormControlLeadingVisual'
+
+export interface Props {
```

Comment:
We typically use `type` instead of `interface` for props.

```suggestion
export type FormControlProps = {
```